src/theme/muiTheme.js

```diff
@@ -36,418 +36,422 @@ export const statusRed = "#FF6766";
+            zIndex: 1,
+            marginTop: "0.5em",
+            backgroundColor: "#EEEEEE",
+          },
         },
-        leftStripe: {
-            color: 'white',
-            display: 'flex',
-            alignItems: 'center',
-            justifyContent: 'center',
-            margin: '0 auto',
-            height: '100%',
-            borderTopLeftRadius: '8px',
-            borderBottomLeftRadius: '8px',
+        "& + $selected:before": {
+          opacity: 0,
         },
-        cardRight: {
-            width: '100%',
-            height: '100%',
-            background: '#FFFFFF',
-            boxShadow: '0px 0px 8px rgba(196, 196, 196, 0.6)',
-            borderTopRightRadius: '8px',
-            borderBottomRightRadius: '8px',
+        "&:hover": {
+          "&:not($selected)": {
+            backgroundColor: skyBlue,
+          },
+          "&::before": {
+            opacity: 0,
+          },
+          "& + $root:before": {
+            opacity: 0,
+          },
         },
-        accountInfo: {
-            display: 'flex',
-            alignItems: 'center',
-            justifyContent: 'flex-start',
+        "&$selected": {
+          backgroundColor: skyBlue,
+          color: omouBlue,
+          background: "#EBFAFF",
+          borderTop: "1px solid #43B5D9",
+          borderLeft: "1px solid #43B5D9",
+          borderRight: "1px solid #43B5D9",
+          boxSizing: "border-box",
+          borderRadius: "5px 5px 0px 0px",
         },
-        iconStyles: {
-            display: 'flex',
-            alignItems: 'center',
-            justifyContent: 'center',
+      },
+      selected: {
+        backgroundColor: skyBlue,
+        color: omouBlue,
+        "& + $root": {
+          zIndex: 1,
         },
-        cardActions: {
-            display: 'flex',
-            alignItems: 'center',
-            justifyContent: 'space-evenly',
-            marginTop: '10px',
+        "& + $root:before": {
+          opacity: 0,
         },
+      },
+      wrapper: {
+        zIndex: 2,
+        marginTop: "0.5em",
+        textTransform: "initial",
+      },
+      indicator: {
+        display: "none",
+        marginTop: "1.1em",
+      },
     },
-    props: {
-        MuiCardHeader: {
-            titleTypographyProps: {
-                variant: 'h4',
-            },
-            subheaderTypographyProps: {
-                variant: 'body2',
-            },
-        },
+    MuiStepIcon: {
+      text: {
+        fill: white,
+      },
+    },
+    MuiTableHead: {
+      root: {
+        color: omouBlue,
+      },
+    },
+    TableHeadSecondary: {
+      color: charcoal,
+    },
+    MuiTableRow: {
+      root: {
+        color: "inherit",
+        height: "48px",
+      },
+    },
+    //Must use disable typography in the tag on your component
+    MuiDialogTitle: {
+      root: {
+        fontSize: "16px",
+        fontWeight: "500",
+        fontFamily: "Roboto",
+        lineHeight: "16px",
```

Comment:
since we have an h4 const at the top, can we do `root: { ...h4 }` instead?

Reply:
Yes! I had tried to do it like that before and it didn't work... I didn't try using the ... spreader though! It's been updated :)